mod-source-record-storage-server/src/main/java/org/folio/services/handlers/AbstractPostProcessingEventHandler.java

```diff
@@ -257,7 +257,7 @@ private void executeHridManipulation(Record record, JsonObject externalEntity) {
     setExternalIds(externalIdsHolder, externalId, externalHrid);
     boolean isAddedField = addFieldToMarcRecord(record, TAG_999, 'i', externalId);
     if (isHridFillingNeeded()) {
-      fillHrIdFieldInMarcRecord(Pair.of(record, externalEntity));
+      fillHrIdFieldInMarcRecord(Pair.of(record, externalEntity), false);
```

Comment:
@Aliaksandr-Fedasiuk  If this method always uses with "false" argument - maybe just remove this argument and rename method? What do you think?

Reply:
Ready. Thank you